### PR TITLE
docs: add time ordering restriction on long_windows deployment

### PR DESCRIPTION
**Pull request by Jake-00**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
I haved updated DEPLOY_STATEMENT.md among two language version.

* **What is the current behavior?** (You can also link to an open issue here)
https://github.com/4paradigm/OpenMLDB/issues/2534

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

**Comment by Jake-00** [comment]
Hi @zhanghaohit
plz review this pr

File: docs/en/reference/sql/deployment_manage/DEPLOY_STATEMENT.md [hunk 1]
```diff
@@ -146,6 +146,8 @@ The current long window optimization has the following limitations:
 
      - The `<column ref>` should not be `date` type or timestamp.
 
+- It requires the data is loaded in the order of timestamp column in order to get the best performance boost.
```
Comment (zhanghaohit) [inline docs/en/reference/sql/deployment_manage/DEPLOY_STATEMENT.md]: `in the increasing order of the timestamp column`

File: docs/zh/reference/sql/deployment_manage/DEPLOY_STATEMENT.md [hunk 2]
```diff
@@ -144,6 +144,8 @@ interval_literal ::= int_literal 's'|'m'|'h'|'d'
 
      - where 关联的列 `<column ref>`，数据类型不能是 date 或者 timestamp
 
+- 为了得到最佳的性能提升，数据需按 `timestamp` 类型（时间戳）列的顺序加载。
```
Comment (zhanghaohit) [inline docs/zh/reference/sql/deployment_manage/DEPLOY_STATEMENT.md]: `数据需按 `timestamp` 列的递增顺序导入。`

**Review by zhanghaohit — CHANGES_REQUESTED** [review]

**Review by zhanghaohit — APPROVED** [review]

File: docs/zh/reference/sql/deployment_manage/DEPLOY_STATEMENT.md (hunk 2 above)
Comment (Jake-00) [reply docs/zh/reference/sql/deployment_manage/DEPLOY_STATEMENT.md]: thanks for your prompt and constructive review

**Comment by Jake-00** [comment]
hi @aceforeverd
I have implemented the requested changes.  Hope that helps : )

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**